install-deps.sh

```diff
@@ -357,28 +357,26 @@ else
 	fi
         case "$ID" in
             fedora)
-                if test $yumdnf = yum; then
-                    $SUDO $yumdnf install -y yum-utils
-                fi
+                $SUDO $yumdnf install -y $yumdnf-utils
                 ;;
             centos|rhel|ol|virtuozzo)
                 MAJOR_VERSION="$(echo $VERSION_ID | cut -d. -f1)"
-                $SUDO yum install -y yum-utils
+                $SUDO $yumdnf install -y $yumdnf-utils
                 if test $ID = rhel ; then
                     $SUDO yum-config-manager --enable rhel-$MAJOR_VERSION-server-optional-rpms
                 fi
                 rpm --quiet --query epel-release || \
-		    $SUDO yum -y install --nogpgcheck https://dl.fedoraproject.org/pub/epel/epel-release-latest-$MAJOR_VERSION.noarch.rpm
+		    $SUDO $yumdnf -y install --nogpgcheck https://dl.fedoraproject.org/pub/epel/epel-release-latest-$MAJOR_VERSION.noarch.rpm
```

Comment:
unrelated to this patch, but unsure why we do nogpgcheck...

Reply:
Hey @mykaul It's because the key that signs the package is actually shipped within it and we actually install the key on the next line.
```

# wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
# rpm -K epel-release-latest-7.noarch.rpm
epel-release-latest-7.noarch.rpm: RSA sha1 ((MD5) PGP) md5 NOT OK (MISSING KEYS: (MD5) PGP#352c64e5)
# rpm -ivh epel-release-latest-7.noarch.rpm
# rpm -K epel-release-latest-7.noarch.rpm
epel-release-latest-7.noarch.rpm: RSA sha1 ((MD5) PGP) md5 NOT OK (MISSING KEYS: (MD5) PGP#352c64e5)
# rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
# rpm -K epel-release-latest-7.noarch.rpm
epel-release-latest-7.noarch.rpm: rsa sha1 (md5) pgp md5 OK
# rpm -q gpg-pubkey --qf '%{name}-%{version}-%{release} --> %{summary}\n'                                                                                                                                               
gpg-pubkey-f4a80eb5-53a7ff4b --> gpg(CentOS-7 Key (CentOS 7 Official Signing Key) <security@centos.org>)                                                                                                                                     
gpg-pubkey-352c64e5-52ae6884 --> gpg(Fedora EPEL (7) <epel@fedoraproject.org>)
# rpm -qf /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
epel-release-7-12.noarch
```